index.html

```diff
@@ -0,0 +1,846 @@
+        </h2>
+      </a>
+    </div>
+
+    <div class="post-arrow">
+      <a href="#">&#8594; </a>
+    </div>
+  </article>
+
+  <article class="post-wrapper">
+    <div class="post-intro">
+      <a href="#">
+        <span>4 days ago</span>
+      </a>
+      <a href="#">
+        <h2 class="post-title">
+          Different Ways To Design Different Products Pages
+        </h2>
+      </a>
+      <a href="">
+        <span class="post-author">By Suzan Scaca</span> <hr>
+      </a>
+      <a href="#">
+        <h2>
+          By designing digital products for smart phone users who are in there early discivery and research phase,
+          your clients can make more money off of their Products. Today we will look at how it is done.
+        </h2>
+      </a>
+    </div>
+
+    <div class="post-arrow">
+      <a href="#">&#8594; </a>
+    </div>
+  </article>
+
+  <article class="post-wrapper">
+    <div class="post-intro">
+      <a href="">
+        <span>4 days ago</span>
+      </a>
+      <a href="#">
+        <h2 class="post-title">
+          Different Ways To Design Different Products Pages
+        </h2>
+      </a>
+      <a href="">
+        <span class="post-author">By Suzan Scaca</span> <hr>
+      </a>
+      <a href="">
+        <h2>
+          By designing digital products for smart phone users who are in there early discivery and research phase,
+          your clients can make more money off of their Products. Today we will look at how it is done.
+        </h2>
+      </a>
+    </div>
+
+    <div class="post-arrow">
+      <a href="#">&#8594; </a>
+    </div>
+  </article>
+</section>
+
+<!-- New on Smashing TV -->
+<section class="smashing">
+
+  <article class="post-wrapper">
+    <div class="post-intro">
+      <a href="">
+        <span>4 days ago</span>
+      </a>
+      <a href="">
+        <h1 class="post-title">
+          Different Ways To Design Different Products Pages
+        </h1>
+      </a>
+      <a href="">
+        <span class="post-author">By Suzan Scaca</span> <hr>
+      </a>
+      <a href="">
+        <h2>
+          By designing digital products for smart phone users who are in there early discivery and research phase,
+          your clients can make more money off of their Products. Today we will look at how it is done.
+        </h2>
+      </a>
+    </div>
+
+    <div class="post-arrow">
+      <a href="#">&#8594; </a>
+    </div>
+  </article>
+
+  <div class="smashing-main">
+    <a href="" class="image-link"><img src="images/singing-barista.svg" alt="singing-barista"></a>
+    
+    <a href="#">
+      <h1>1,699 Smashing Members</h1>
+    </a>
+    <h2>A freindluy community that keeps this website alive. With books, webinars, discounts and networking
+      - for just 1 coffee a month. <a href="#">Join the community</a>
+    </h2>
+    <div class="features-link">
+      <a href="#" class="explore">Explore Features</a>
+      <span>&#8594;</span> 
```

Comment:
The <span> tag is an inline container used to mark up a part of a text, or a part of a document. The correct use of span is as follows:
```
<p>My mother has <span style="color:blue">blue</span> eyes.</p>
```